keymap_drawer/config.py

```diff
@@ -127,6 +131,12 @@ class Config:  # pylint: disable=missing-class-docstring
         """
     )
 
+    glyph_tap_size: int = 12
+    glyph_hold_size: int = 10
+    glyph_shift_size: int = 8
```

Comment:
```suggestion
    glyph_shifted_size: int = 8
```
Small change for naming consistency.